ReactNativeClient/android/app/build.gradle

```diff
@@ -180,6 +180,7 @@ android {
 }
 
 dependencies {
+    implementation project(':@react-native-community_geolocation')
 	implementation "org.webkit:android-jsc:r241213"
```

Comment:
As we're using RN 60, this shouldn't be necessary. Could you confirm if it works without it please?

Reply:
It does, removed it.